recipe/meta.yaml

```diff
@@ -48,7 +48,25 @@ test:
 about:
   home: http://nuitka.net
   license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE.txt
   summary: Python compiler with full language support and CPython compatibility
+  description: |
+    Nuitka is the Python compiler. It is a seamless replacement or extension
+    to the Python interpreter and compiles every construct that CPython 2.6,
+    2.7, 3.2, 3.3, 3.4, 3.5, and 3.6 have. It then executes uncompiled code,
```

Comment:
Reminds me that I have recently removed "3.2" support. I need to find the space where you took that
from and update it.

Reply:
I took it from the README.rst overview: https://github.com/kayhayen/Nuitka/blob/develop/README.rst#overview